code/modules/reagents/Chemistry-Reagents/Other-Reagents.dm

```diff
@@ -175,8 +175,7 @@
 	if(!istype(M, /mob/living))
 		return
 	if(method == TOUCH)
-		M.adjust_fire_stacks(-(reac_volume))
-	..()
+		M.ExtinguishMob()
```

Comment:
This is too overpowered. I think the best thing is to multiply the reac_volume by 2 or 3 so that water is much more effective